plugins/plugin-js/src/main/kotlin/com/kotlin/aws/js/runtime/utils/Groups.kt

```diff
@@ -0,0 +1,6 @@
+package com.kotlin.aws.js.runtime.utils
+
+object Groups {
+    const val aws = "aws"
```

Comment:
I would expect that groups would be someting like `js` or `js runtime`

Reply:
changed to js